_docs/110_faq.md

```diff
@@ -112,19 +112,20 @@ you when you clone your repository.
 
 ### Should I `yadm add` encrypted files to repository?
 
-No, you should not. Files you want encrypted should be added to the file
-`.config/yadm/files.gpg` using the `yadm encrypt` command. Then
-`.config/yadm/files.gpg` should be added to the yadm repository. This way, only
-an encrypted collection of those files are put into the repository. After
-cloning or updating your repository, you can use `yadm decrypt` to extract those
-files from `.config/yadm/files.gpg`. See the
+No, you should not. Files you want encrypted should be added to the text file
+`.config/yadm/encrypt`. Then, use the `yadm encrypt` command to encrypt the listed 
+files, and add them to `~/.local/share/yadm/archive`. 
+Then `~/.local/share/yadm/archive` should be added to the yadm repository. This way, 
+only encrypted versions of those files are added to the repository. After
+cloning or updating your repository, you can use `yadm decrypt` to extract the
+encrypted files in `~/.local/share/yadm/archive`. See the
```

Comment:
"in" would read better as "from", I think.

Reply:
Agreed